spec/jobs/scheduled_publishing_job_spec.rb

```diff
@@ -23,7 +23,7 @@
         ScheduledPublishingJob.perform_now(edition.id)
       end
 
-      it "calls ScheduledPublishMailer for each editor of the edition" do
+      it "calls ScheduledPublishMailer's success_email method for each editor of the edition" do
```

Comment:
Minor, but I think we can afford to be more concise here. How about 'delegates to the ScheduledPublishMailer for each editor'